firebase-firestore/src/main/java/com/google/firebase/firestore/local/MutationQueue.java

```diff
@@ -47,8 +47,17 @@
   /** Sets the stream token for this mutation queue. */
   void setLastStreamToken(ByteString streamToken);
 
-  /** Creates a new mutation batch and adds it to this mutation queue. */
-  MutationBatch addMutationBatch(Timestamp localWriteTime, List<Mutation> mutations);
+  /**
+   * Creates a new mutation batch and adds it to this mutation queue.
+   *
+   * @param localWriteTime The original write time of this mutation.
+   * @param baseMutations Mutations that are used to populate the base values when this mutation is
+   *     applied locally. These mutations are used to locally overwrite values that are persisted in
+   *     the remote document cache.
```

Comment:
Consider revising this comment to (at least somewhat) match the proto changes, etc. too?  (and rename baseMutations => baseStateMutations)